paper/paper.md

```diff
@@ -0,0 +1,138 @@
+A collection of predefined simulation files can be found in the `examples` directory.
+
+### Feature highlights
+
+So far, the following feature highlights have been implemented:
+
+* *Fluid Systems*
+    + `WeaklyCmpressibleSPHSystem` (WCSPH): Standard SPH method originally developed by [@Monaghan:1977] to simulate astrophysics applications.
+    + `EntropicallyDampedSPHSystem` (EDAC): As opposed to the WCSPH scheme, which uses an equation of state, this scheme uses a pressure evolution equation to calculate the pressure which is derived by [@Clausen:2013] and adapted to SPH by [@Ramachandran:2019].
+
+* *Structure Systems*
+    + `TotalLagrangianSPHSystem` (TLSPH): System to simulate elastic structure where the interaction is defined with stress tensors [@O_Connor:2021].
+    + `DEMSystem`: Discrete element system [@Bicanic:2004], [@Cundall:1979].
+
+* *Boundary Systems*
+    + `BoundarySPHSystem` with several boundary models where each model follows a different interaction rule.
+    + `OpenBoundarySPHSystem`: System to simulate non-reflecting (open) boundary conditions [@Lastiwka:2009]
+
+* *Correction methods and models*
+  + Density diffusion [@Antuono:2010]
+  + Transport-velocity formulation (TVF) [@Adami:2013]
+  + Intra-particle-force surface tension [@Akinci:2013]
+  + Kernel corrections TODO: refs
+
+* *TODO: category naming?*
+  + Efficient grid neighborhood search
+  + GPU support
+
+\autoref{fig:falling_sphere} illustrates an example of our simulation results. In this example, an elastic sphere modeled with TLSPH falls into a tank filled with water, modeled by WCSPH.
+
+![Elastic sphere falling into a tank filled with water. \label{fig:falling_sphere}](falling_sphere_combined_nonstick_4k_178.png){width=100%}
+
+The current state allows also to validate our simulation and produce quantitative results with a post-process callback. \autoref{fig:beam_y_deflection} shows simulation results of TrixiParticles.jl (on the left) and [@O_Connor:2021] (on the right) compared against a reference value of [@Turek:2007].
+The curves show the y-deflection of the tip of a beam oscillating under its own weight.
+The results obtained with TrixiParticles.jl perfectly match those of [@O_Connor:2021].
+
+![Comparison of TrixiParticles.jl and  [@O_Connor:2021] against [@Turek:2007]: Tip y-deflection of an oscillating beam with different resolutions, where $t_s$ is the thickness of the beam and $dp$ is the particle spacing. \label{fig:beam_y_deflection}](oscillating_beam.png){width=100%}
+
+As a young project that aims to be the Julia code for particle-based simulation we expect to integrate other methods such as incompressible SPH (ISPH) [@Liu:2022] or heat conduction with SPH [@Biriukov:2018] and couple them among each other and even with mesh-based methods.
```

Comment:
```suggestion
As a young project that aims to be the Julia code for particle-based simulation we expect to integrate other methods such as incompressible SPH (ISPH) [@Liu:2022] or heat conduction with SPH [@Biriukov:2018], and DEM specifically tailored for applications in coatings [@Ghasemi:2019] [@Laukannen:2020]. 
```
https://doi.org/10.1016/j.surfcoat.2018.11.068
https://doi.org/10.1016/j.triboint.2020.106534

Reply:
Forgot the second part:
In the future these methods will be coupled with each other and even with mesh-based methods to provide a comprehensive framework.